allensdk/brain_observatory/behavior/session_apis/data_transforms/behavior_data_xforms.py

```diff
@@ -150,6 +32,22 @@ def _behavior_stimulus_file(self) -> pd.DataFrame:
         """
         return pd.read_pickle(self.get_behavior_stimulus_file())
 
+    def get_behavior_session_uuid(self) -> Optional[int]:
+        """Get the universally unique identifier (UUID) number for the
+        current behavior session.
+        """
+        data = self._behavior_stimulus_file()
+        behavior_pkl_uuid = data.get("session_uuid")
+        # Sanity check to ensure that pkl file data matches up with
+        # the behavior session that the pkl file has been associated with.
+        assert_err_msg = (
+            f"The behavior session UUID ({behavior_pkl_uuid}) in the behavior "
+            f"stimulus *.pkl file ({self.get_behavior_stimulus_file()}) does "
+            f"does not match the LIMS UUID ({self.foraging_id}) for "
```

Comment:
I don't see these values in the class (self.foraging_id, self.behavior_session_id, etc.)